src/cmd_argument.rs

```diff
@@ -268,3 +268,24 @@ impl CmdArgument for &Path {
         self.to_path_buf().prepare_config(config);
     }
 }
+
+/// See the [`CmdArgument`] implementation for [`Stdin`] below.
+pub struct Stdin<'a>(pub &'a str);
+
+/// Passes the given [`&str`] into the child on `stdin`.
```

Comment:
Not sure how to word this, but something that makes it clear that `Stdin` can be used multiple times might be good:

```suggestion
/// Writes the given [`&str`] to the child's standard input.  If `Stdin` is used multiple times,
/// each string will be written written to the child's standard input in the order it appears.
```

Reply:
What do you think about [this](https://github.com/soenkehahn/cradle/pull/59/commits/31592105af5fe5eea574a5a0287f5eeaa65a9aac#diff-314b0e8d3543865f2dac2e6e8ae1d3cc4abb8dadebf2a4576d8674c07c010ea4R275-R277)?

Reply:
SGTM

Reply:
SGTM